core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/config/UserStoreConfigXMLProcessor.java

```diff
@@ -494,7 +495,7 @@ private CipherHolder cipherTextToCipherHolder(byte[] cipherText) {
      * IMPORTANT: this is copy of org.wso2.carbon.core.util.CipherHolder, what ever changes applied here need to update
      *              on above
      */
-    private class CipherHolder {
+    private class CipherMetaDataHolder {
```

Comment:
Why we needed another meta data holder defined here.

Reply:
Fixed with 39950c6a8d10d5c5723a817db9e0a05d0662761b